Source/Dafny/Compilers/Compiler-python.cs

```diff
@@ -1477,9 +1473,9 @@ protected class ClassWriter : IClassWriter {
 
       try {
         using var pythonProcess = Process.Start(psi);
-        pythonProcess.StandardInput.Close();
-        pythonProcess.WaitForExit();
-        return pythonProcess.ExitCode == 0;
+        pythonProcess?.StandardInput.Close();
```

Comment:
Is this a case of `Process.Start` declaring a return type of `Process?` when it should be `Process`? If so, it would be good to explicitly cast to a non-null type once instead.

Otherwise, I would think we'd want to propagate an error if we didn't actually start the process.